src/collectd.conf.pod

```diff
@@ -139,6 +140,13 @@ defined in this plugin. By default, this is disabled.
 
 Specifies the value of the timeout argument of the flush callback.
 
+=item B<StartRead> I<Seconds>
+
+Sets a start time for the specific read plugin. The value is interpreted as 
+minutes, seconds and milliseconds of an hour. This plugin setting overrides the
```

Comment:
1) "milliseconds of an hour" is strange.
2) milliseconds precision has no sence

`DelayRead - specifies plugin-specific delay before first metrics collection. This overrides the global DelayRead setting.`

Don't copy-n-paste text blocks you don't understand fully. There is no way for plugin to specify start time now and in your patch.